src/main/java/com/example/java/maven/cardGamePeterPan/Player.java

```diff
@@ -0,0 +1,86 @@
+package com.example.java.maven.cardGamePeterPan;
+
+import java.util.ArrayList;
+
+public class Player {
+    private final String name;
+    private final ArrayList<Card>hand;
```

Comment:
use interface: `List` instead of specific implementation: `ArrayList`

Reply:
Done